packages/rpc-graphql/src/schema/types.ts

```diff
@@ -32,6 +54,10 @@ export const commonTypeDefs = /* GraphQL */ `
         rewardType: String
     }
 
+    scalar Signature
+
+    scalar Slot
```

Comment:
<!--__GRAPHITE_HTML_TAG_START__--><i class='graphite__hidden'>[Re: lines 9 to 59]<br/><br/></i>

<!--__GRAPHITE_HTML_TAG_END__-->![image.png](https://graphite-user-uploaded-assets-prod.s3.amazonaws.com/lE1zD7Jpg6mWNv3djNKZ/df557ac3-d50f-48a3-a998-60c419255c5e.png)

<!--__GRAPHITE_HTML_TAG_START__--><span class='graphite__hidden'><br/><br/>See this comment inline on <a href="https://app.graphite.dev/github/pr/solana-labs/solana-web3.js/2095?utm_source=unchanged-line-comment">Graphite</a>.</span>

<!--__GRAPHITE_HTML_TAG_END__-->